server/auth/local/verificationToken.model.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import {Schema} from 'mongoose';
+import uuid from 'node-uuid';
+var mongoose = require('bluebird').promisifyAll(require('mongoose'));
+
+// Verification token model
+var verificationTokenSchema = new Schema({
+    userId: {
+      type: Schema.Types.ObjectId,
+      required: true,
+      ref: 'User'
+    },
+    token: {
+      type: String,
+      required: true
+    },
+    createdAt: {
```

Comment:
What happens after expiring? Do we re-issue a token? If so how, else do we delete the account? Also, the user is not told about the time limit.